src/core/modules/listeners/listeners_manager.cpp

```diff
@@ -149,7 +161,38 @@ object CListenerManager::__getitem__(unsigned int index)
 
 void CListenerManager::clear()
 {
-	m_vecCallables.RemoveAll();
+	if (GetCount()) {
+		m_vecCallables.RemoveAll();
+		Finalize();
+	}
+}
+
+
+//-----------------------------------------------------------------------------
+// Convar changed callback.
+//-----------------------------------------------------------------------------
+void ConVarChangedCallback(IConVar* var, const char* pOldValue, float flOldValue)
+{
+	CConVarChangedListenerManager* on_convar_changed_listener_manager = GetOnConVarChangedListenerManager();
```

Comment:
This could be made static so that it is only retrieved once.

Reply:
Done!